modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -147,6 +147,17 @@ extension KeyedEncodingContainerProtocol {
         }
     }
 
+	{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} mutating func encode(_ value: Decimal, forKey key: Self.Key) throws {
+        var mutableValue = value
+        let stringValue = NSDecimalString(&mutableValue, Locale(identifier: "en_US"))
```

Comment:
This is not for displaying the value to the user but to convert the decimal back to a JSON string. I don‘t think `Locale.current` whould be right in that case.

Reply:
But is `Locale(identifier: "en_US")` the right `locale` to use outside of US?

Reply:
This fix is for properties of type „string“ / format „number“ which seems to be the „correct“ definition to transmit a decimal. And since numbers are transmitted with a locale which has a dot as decimal separator in most technical applications and the spring boot server is used for testing as a WebService does it the same way, i whould argue that an hard coded english locale is the right choice in this situation. If i change it to `Locale.current` it simply stop working in my german locale for example.